packages/nx/src/utils/output.ts

```diff
@@ -1,12 +1,13 @@
-import * as chalk from 'chalk';
+import * as pc from 'picocolors';
 import { EOL } from 'os';
 import * as readline from 'readline';
-import { isCI } from './is-ci';
 import type { TaskStatus } from '../tasks-runner/tasks-runner';
 
 const GH_GROUP_PREFIX = '::group::';
 const GH_GROUP_SUFFIX = '::endgroup::';
 
+const orangeText = (str: string) => `\x1B[38;5;214m${str}\x1B[39m`;
```

Comment:
we only have one occurrence of a non-standard colour (i.e. an rgb term colour). so I went ahead and just wrote this function for it (we did similar in storybook as it was the same situation, one colour)

let me know if this is ok. we could also just change to a basic colour instead of orange

Reply:
i did a search of `chalk.keyword('orange')`, i think there are a few places outside nx packages also use color orange.